deployments/iaa_plugin/overlays/iaa_initcontainer/iaa_initcontainer.yaml

```diff
@@ -13,7 +13,7 @@ spec:
               fieldRef:
                 fieldPath: spec.nodeName
           - name: IDXD_DEVICE_TYPE
-            value: "iaa"
+            value: "iax"
```

Comment:
It looks this is just a device identifier that needs to be in sync with the json conf file name. Is there a reason for renaming?

Reply:
This way it's in sync with json and idxd-init.sh